src/dotnet/scripts/dotnet-helpers.sh

```diff
@@ -117,3 +115,22 @@ install_runtime() {
         --install-dir "$DOTNET_INSTALL_DIR" \
         --no-path
 }
+
+# Installs a .NET workload
+# Usage: install_workload <workload_id>
+# Reference: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-install
+install_workload() {
+    local workload_id="$1"
+
+    # Prevent "Welcome to .NET" message from dotnet
+    local DOTNET_NOLOGO=true
+
+    # Prevent generating a development certificate
+    local DOTNET_GENERATE_ASPNET_CERTIFICATE=false
+
+    echo "Executing dotnet workload install $workload_id --temp-dir /tmp/dotnet-workload-temp-dir"
+    dotnet workload install "$workload_id" --temp-dir /tmp/devcontainer-nuget-cache
```

Comment:
Suggestion to improve the code: (untested)

```
if ! dotnet workload list | grep -q "$workload_id"; then
    echo "Installing .NET workload: $workload_id"
    dotnet workload install "$workload_id" --temp-dir /tmp/devcontainer-nuget-cache
else
    echo "Workload $workload_id is already installed."
fi
```

Reply:
As I was testing this, I just realized you can pass multiple workloads in one line.

```
dotnet workload install <workload_id_1> <workload_id_2> <workload_id_n>
```

The install command already checks for duplicates, so I think that is the more preferred approach.

Reply:
This is now done

Reply:
🎉